walk-safe/src/Components/App/App.js

```diff
@@ -1,16 +1,12 @@
 import React from 'react';
-import Login from '../Login/Login';
+// import Login from '../Login/Login';
 import Form from '../Form/Form';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>
-          WALK SAFE
-        </h1>
-      </header>
-      <Login />
+      {/* <Login /> */}
```

Comment:
We may want to look at adding a navigation component, now that we have Main, AddContact, and an upcoming CurrentTrip component